src/order/sup_indep.lean

```diff
@@ -169,15 +178,15 @@ theorem set_independent.mono {t : set α} (hst : t ⊆ s) :
 λ a ha, (hs (hst ha)).mono_right (Sup_le_Sup (diff_subset_diff_left hst))
 
 /-- If the elements of a set are independent, then any pair within that set is disjoint. -/
-lemma set_independent.disjoint {x y : α} (hx : x ∈ s) (hy : y ∈ s) (h : x ≠ y) : disjoint x y :=
-disjoint_Sup_right (hs hx) ((mem_diff y).mpr ⟨hy, by simp [h.symm]⟩)
+lemma set_independent.pairwise_disjoint : s.pairwise disjoint :=
```

Comment:
```suggestion
lemma set_independent.pairwise_disjoint : s.pairwise_disjoint id :=
```

Reply:
This looks worse to me, as it means `h.pairwise_disjoint ha hb hab` has type `(disjoint on id) a b` which is syntactically much more annoying than `disjoint a b`.

Reply:
I've made this change a bunch of times #9898 and it just works. I know you don't like `set.pairwise_disjoint` but this allows dot notation and a unified interface.

Reply:
Would you mind making a PR to note and justify this choice in the docstring for `set.pairwise_disjoint`? A sentence or two saying what on mobile I'll summarize as "prefer `pairwise_disjoint id` to `pairwise disjoint` because dots despite unfolding" would go a long way

Reply:
#12878